hi-ml-cpath/src/health_cpath/utils/output_utils.py

```diff
@@ -205,7 +206,12 @@ def should_save_validation_outputs(self, metrics_dict: Mapping[MetricsKey, Metri
         :return: Whether this is the best validation epoch so far.
         """
         # The metric needs to be computed on all ranks to allow synchronisation
-        metric_value = float(metrics_dict[self.primary_val_metric].compute())
+        metric = metrics_dict[self.primary_val_metric]
+        if not metric._update_called:
+            return False
+        metric_value = float(metric.compute())
+        if isinstance(metric, Accuracy):
```

Comment:
Would you mind adding a comment explaining why we need to special-case for `Accuracy`?